### ISSUE TITLE
Visual Studio 2015 Cookbook - Typescript 2

### ISSUE THREAD

**Issue by sam-s4s** [angular/angular.io#3102]
In the "Visual Studio 2015 Cookbook", under the heading "Prerequisite: Install TypeScript 2 for Visual Studio 2015", there is a link to download "TypeScript 2.0 for Visual Studio 2015".

This installer completely broke TS support for VS2015 for me. However, there is a newer version of this file - confusingly with the same file name... but it actually fixed all my problems (I was having some rxjs problems, all fixed now).

Old (broken) installer...
http://download.microsoft.com/download/6/D/8/6D8381B0-03C1-4BD2-AE65-30FF0A4C62DA/TS2.0.3-TS-release20-nightly-20160921.1/TypeScript_Dev14Full.exe

New (fixed) installer...
http://download.microsoft.com/download/6/D/8/6D8381B0-03C1-4BD2-AE65-30FF0A4C62DA/TS2.1-dev14update3-20170111.1/TypeScript_Dev14Full.exe

I think this should be updated so others can avoid my pain :P

**Comment by wardbell** [comment]
We probably should be referring to the Microsoft web site, https://www.microsoft.com/en-us/download/details.aspx?id=48593, rather than a specific download.

My current worry, however, is that the specific download you have selected is for TS 2.1 and, AFAIK, Angular apps can't use that version of TS yet.  Checking on that now. Then will take appropriate action

Thanks for raising the issue.

**Comment by sam-s4s** [comment]
Yes, probably correct about linking to the page instead of the direct download.

Ah ok! Yeah, I'm not sure why the older version wrecked VS2015 so much... seems strange!

I assume backwards compatibility (in that even TS1 apps still seem to be fine in VS2015 after applying TS2.1) - so surely whatever version Angular happens to be using should be covered also?

(we have 3 different versions of TS on the go here in various projects - all developers have updated to the new TS 2.1 for VS2015 and none have encountered any problems yet)
